src/cli/Cli.ts

```diff
@@ -151,13 +153,26 @@ export class Cli {
         }
       }
     }
+    const shouldPrompt = this.getSettingWithDefaultValue<string | boolean>(settingsNames.prompt, false);
```

Comment:
`shouldPrompt` is a boolean and we treat it as boolean in our code, so let's remove the `string` type